### DISCUSSION TITLE
Install of 23.1.4 Does Not Replace 23.1.3

### DISCUSSION THREAD

**Discussion by CenturySparkle** [dbeaver/dbeaver#21014] [Q&A] [answered]
### Description

I just recently installed 23.1.3 and now there is an updated to 23.1.4.  I have allowed DBeaver to update itself twice now and both times I see it downloads and installs 23.1.4 but when I run DBeaver after the install it is still running 23.1.3.

### DBeaver Version

23.1.3

### Operating System

Windows 11 22H2

### Database and driver

N/A

### Steps to reproduce

Open DBeaver.  See the Update Notice window announcing version 23.1.4.  Click the Update button.  Allow the installation run to completion.  Start DBeaver, notice that it is still version 23.1.3.

**Comment by E1izabeth** [comment, marked as answer]
Please check that you choose the same setting about installation for current user or globally while installing new DBeaver. Most probably the old DBeaver isn't replaced because of that.
